uyuni-docs-helper

```diff
@@ -4,20 +4,20 @@
 # under the terms of the GNU General Public License as published by the Free
 # Software Foundation, either version 3 of the License, or (at your option)
 # any later version.
-# 
+#
 # This program is distributed in the hope that it will be useful, but WITHOUT
 # ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
 # FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
 # more details.
-# 
+#
 # You should have received a copy of the GNU General Public License along
 # with this program. If not, see <https://www.gnu.org/licenses/>.
 
 # Language (force it so getopt messages are always in english, as the script)
 LANG=en_EN
 
 # Get script name
-SCRIPT=$(basename ${0})
+SCRIPT=$(basename "${0}")
```

Comment:
I guess the tool is being a bit over overzealous, quotes should not be needed in most of the changes it did, are for things that will never have whitespaces or glob patterns.

But since it should not break having it, then I am OK with the changes.